packages/animations/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## NEXT
```

Comment:
Why would we not want to release the updated sample code?

Reply:
pub.dev only shows `lib/main.dart`, and the changes in this package aren't to that file, so publishing wouldn't do anything (unless people are running examples out of their pub cache, but that's not something we expect/support in the repo policy).